experimental/dds/tree2/src/feature-libraries/contextuallyTyped.ts

```diff
@@ -378,7 +411,7 @@ export function cursorForTypedData<T extends AllowedTypes>(
 	data: AllowedTypesToTypedTrees<ApiMode.Simple, T>,
 ): ITreeCursorSynchronous {
 	return cursorFromContextualData(
-		schemaData,
+		{ schema: schemaData, fieldSource: defaultFieldSource },
```

Comment:
```suggestion
		{ schema: schemaData },
```
Now that fieldSource can be undefined, and that ends up being handled the same as having a fieldSource that always returns undefined, I don't think passing `defaultFieldSource` in anywhere is necessary.